spec/features/create_child_work_spec.rb

```diff
@@ -28,6 +28,7 @@
       fill_in('Title', with: work_title)
       click_on('Save')
     end
+
```

Comment:
squash out whitespace-only change?

Reply:
This is likely the "And Jeremy rebased 20 commits into 3" effect.